src/main/java/seedu/address/model/Model.java

```diff
@@ -15,6 +16,10 @@
     /** {@code Predicate} that always evaluate to true */
     Predicate<Person> PREDICATE_SHOW_ALL_PERSONS = unused -> true;
```

Comment:
Would this be required still?

Reply:
it is part of a legacy code so should we delete it?